packages/scheduler/src/__tests__/Scheduler-test.js

```diff
@@ -288,6 +288,26 @@ describe('Scheduler', () => {
     expect(Scheduler).toFlushWithoutYielding();
   });
 
+  it('cancelling the currently running task', () => {
+    const task = scheduleCallback(NormalPriority, () => {
```

Comment:
Are there any other priorities we need to test cancelling on other than normal? Should there be a test to confirm that discrete priority events do not get cancelled?

Reply:
The priority shouldn't be relevant, but I'll add the same test at user-blocking priority as an extra precaution.